extensions/wikia/CuratedContent/CuratedContentController.class.php

```diff
@@ -444,6 +444,32 @@ public function getCuratedContentQuality() {
 		}
 	}
 
+	public function getData( ) {
+		global $wgWikiaCuratedContent;
+		$data = [];
+
+		foreach ( $wgWikiaCuratedContent as $section ) {
```

Comment:
Should we add a check if it even exists? It will probably throw some fatals for wikis without CC enabled, right?